src/presentation/ui/page/databases.templ

```diff
@@ -0,0 +1,352 @@
+package page
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	"github.com/speedianet/os/src/domain/valueObject"
+	componentDatabases "github.com/speedianet/os/src/presentation/ui/component/databases"
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+	presenterDto "github.com/speedianet/os/src/presentation/ui/presenter/dto"
+)
+
+script DatabasesIndexLocalState() {
+    document.addEventListener('alpine:init', () => {
+		Alpine.data('databases', () => ({
+			// Primary states
+			database: {},
+			databaseUser: {},
+			resetPrimaryStates() {
+				this.database = {
+					'name': '',
+					'size': ''
+				}
+				this.databaseUser = {
+					'username': '',
+					'password': '',
+					'privileges': []
+				}
+			},
+			init() {
+				this.resetPrimaryStates();
+			},
+
+			// Auxiliary states
+			changeSelectedDatabaseType(databaseType) {
+				htmx.ajax(
+					'GET',
+					'/databases/?dbType=' + databaseType,
+					{
+						select: '#databases-page-content',
+						target: '#databases-page-content',
+						swap: 'outerHTML transition:true'
+					},
+				);
+			},
+			installDatabaseService(databaseType) {
+				htmx.ajax(
+					'POST',
+					'/api/v1/services/installables/',
+					{
+						swap: 'none',
+						values: { name: databaseType },
+					},
+				);
+			},
+			get shouldDisableCreateDatabaseSubmitButton() {
+				return this.database.name == ''
+			},
+			get shouldDisableCreateDatabaseUserSubmitButton() {
+				return this.database.name == '' || this.databaseUser.username == '' || this.databaseUser.password == ''
+			},
+			generateRandomUserPassword() {
```

Comment:
You mentioned that you would put this inside the password component. Why didn't you?

Reply:
I initially planned to make this change only when using the component on the `/accounts/` page, but I've gone ahead and made the adjustment now.